### PR TITLE
fix: issue #16

### PR DESCRIPTION
**Pull request by aryanas159**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: snippets/snippets-ts.json [hunk 1]
```diff
@@ -1 +1,249 @@
-{}
\ No newline at end of file
+{
+	"fastify hello server": {
+		"scope": "javascript,typescript",
+		"prefix": "ffhello",
+		"body": [
+			"const app = require('fastify')({ logger: true })",
```
Comment (climba03003) [inline snippets/snippets-ts.json:6]: You should use `import` here.

File: snippets/snippets-ts.json [hunk 2]
```diff
@@ -1 +1,249 @@
-{}
\ No newline at end of file
+{
+	"fastify hello server": {
+		"scope": "javascript,typescript",
```
Comment (climba03003) [inline snippets/snippets-ts.json:3]: The scope should  be `TypeScript` only.

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Comment by aryanas159** [comment]
I have made the required changes.

**Review by climba03003 — APPROVED** [review]

File: snippets/snippets-ts.json [hunk 3]
```diff
@@ -1 +1,250 @@
...
+			"})",
+			"app.post('/', async (request, reply) => {",
+			"  return request.body",
+			"})",
+			"app.listen({ port: 8080 })"
+		],
+		"description": "Create an hello world server"
+	},
+	"fastify register": {
+		"scope": "javascript,typescript",
```
Comment (Eomm) [inline snippets/snippets-ts.json:19]: We should fix all the `scope`s in the file

Comment (climba03003) [reply snippets/snippets-ts.json:19]: Yes

**Comment by aryanas159** [comment]
@Eomm  I have made changes for the scopes, can you please review.

**Review by Eomm — APPROVED** [review]

**Pull request merged**